### PR TITLE
Fix bug in FlowProject.add_operation.

### PR DESCRIPTION
**Pull request by alacour**
## Description
There was a bug in FlowProject.add_operation. I fixed it.

Resolves: #254.

## Types of Changes

- [ ] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac-flow/blob/master/ContributorAgreement.md).
- [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md#code-style) of this project.
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [ ] I have updated the API documentation as part of the package doc-strings.
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [x] I have updated the [changelog](https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt).

### CODE REVIEW

**Comment by csadorf** [comment]
I think it would be good for @b-butler to also have a look at this.

File: changelog.txt [hunk 1]
```diff
@@ -34,6 +34,10 @@ Changed
 +++++++
 - Raises ValueError when --job-name passed by the user because that interferes with status checking (#164, #241).
 
+Fixed
++++++
+- Fix bug in FlowProject.add_operation
```
Comment (vyasr) [inline changelog.txt]: @alacour can you make this entry a bit more descriptive about the bug, and link this pull request (just put the number in parenthesis like in the other changelog entries)?

File: flow/templates/umich-greatlakes.sh [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #SBATCH --nodes={{ nn|default(1, true) }}
 #SBATCH --ntasks-per-node={{ (gpu_tasks, cpu_tasks)|max }}
 #SBATCH --gres=gpu:{{ gpu_tasks }}
+
```
Comment (b-butler) [inline flow/templates/umich-greatlakes.sh]: There shouldn't be an extra line here. Please delete it.

**Comment by b-butler** [comment]
I was the one to find this bug with @alacour. The changes will match the expection that @alacour and others have for the `add_operation` function to create operations that always are executed through the command line.

**Comment by alacour** [comment]
Fixes #255

**Comment by csadorf** [comment]
I don't think we need a changelog entry for a fix that is addressing a non released bug.

File: changelog.txt [hunk 3]
```diff
@@ -34,6 +34,11 @@ Changed
 +++++++
 - Raises ValueError when --job-name passed by the user because that interferes with status checking (#164, #241).
 
+Fixed
++++++
+- Fix bug in FlowProject.add_operation that was calling FlowOperation instead
+of FlowCmdOperation.
```
Comment (bdice) [inline changelog.txt]:
```suggestion
- Fix bug in FlowProject.add_operation that was calling FlowOperation instead of FlowCmdOperation (#254, #255).
```

**Comment by bdice** [comment]
@csadorf I didn't see your comment before I just reformatted the changelog entry, but I agree that it can be removed. I will let @vyasr and @b-butler handle that, since they're assigned as reviewers.

**Comment by b-butler** [comment]
If @vyasr is fine with not adding an entry to the changelog, I am as well.

**Comment by vyasr** [comment]
Agreed.

**Pull request merged**